docs/code-quality/ca1010.md

```diff
@@ -60,6 +60,21 @@ dotnet_code_quality.ca1010.api_surface = private, internal
 
 You can configure this option for just this rule, for all rules, or for all rules in this category (Design). For more information, see [Configure FxCop analyzers](configure-fxcop-analyzers.md).
 
+### Additional required generic interfaces
+
+You can configure the List (separated by '|') of interface names with their required generic fully qualified interface (separated by '->').
```

Comment:
Shouldn't this be:
```suggestion
You can configure the List of interface names (separated by '|') with their required generic fully qualified interface (separated by '->').
```

I'll let the doc experts confirm the correct sentence formation.

Reply:
Yes, I'll commit that.